src/robusta/core/sinks/datadog/datadog_sink.py

```diff
@@ -28,9 +28,10 @@
 
 
 class DataDogSink(SinkBase):
-    def __init__(self, sink_config: DataDogSinkConfigWrapper, cluster_name: str):
-        super().__init__(sink_config.datadog_sink)
-        self.cluster_name = cluster_name
+    def __init__(self, sink_config: DataDogSinkConfigWrapper, registry):
+        super().__init__(sink_config.datadog_sink, registry)
+        global_config = self.registry.get_global_config()
```

Comment:
We're repeating these lines few times.
Wouldn't it be easier to set these 3 attributes in the SinkBase, and access it where ever needed in the inheriting classes?

Reply:
good idea